client/src/components/ui/navigationbar/TopBar.svelte

```diff
@@ -1,39 +1,76 @@
 <script lang="ts">
     import type { User } from '../../../../../model/src/user.ts';
 
-    import Hamburger from '../../icons/Menu.svelte';
+    import Hamburger from '../../icons/Hamburger.svelte';
 
     export let show = false;
     export let user: User;
+
 </script>
 
-<nav on:click|stopPropagation on:keypress>
-    <div>
-        <span id="icon"><Hamburger alt="Expand Menu" on:click={() => (show = !show)} /></span>
-        <span>Hello, {user.name}!</span>
-    </div>
-    <img src={user.picture} alt="{user.name}" />
+<nav id="navcontainer" on:click|stopPropagation on:keypress>
+    <nav id="leftnav">
+        <span id="icon"><Hamburger bind:open={show} on:click={() => {show = !show;}} /></span>
+        <!-- TODO: Temporarily disabled if officeList is null -->
+        <!-- <OfficeSelect offices={$officeList} bind:oid={selectedId} /> -->
+    </nav>
+    <p class="doctrack">DocTrack</p>
+    <nav id="profilenav">
+        <span class="unselectable">{user.name} </span>
+        <span><img class="unselectable" src={user.picture} alt="{user.name[0]}" /></span>
+    </nav>

```

Comment:
Having multiple nested `nav` defeats the purpose of semantic HTML. We should reconsider the structure here.